index.html

```diff
@@ -332,6 +332,24 @@ <h2>Content for Step 3</h2>
           <sgds-button refId="myStepper" methodType="last" variant="warning">Go to last page</sgds-button>
           <sgds-button refId="myStepper" methodType="first" variant="danger">Go to first page</sgds-button>
         </div>
+        <div class="container">
+            <h2>File Upload</h2>
+            <div class="row">
+                <div class="col is-6">
+                    <sgds-fileupload id="my-fileupload" checkedIcon='<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-check-all align-middle me-2" viewBox="0 0 16 16">
```

Comment:
might want to handle the class "align-middle me-2" inside fileupload component if its part of the design.  high chance user wont bother adding this css classes themselves.

Reply:
either handle it in scss file like write the styling for the svg or use the bootstrap css align-middle me-2, maybe writing the scss is easier than trying to inject the classes into the svg?  u can try see which is more straightforward